MinPlatformPkg/Include/Library/TestPointMtrrInfoLib.h

```diff
@@ -0,0 +1,39 @@
+/** @file
+TestPointMtrrInfoLib.h
+
+An interface for platforms to define the expected MTRR cache types for specific
+regions by Ready To Boot.  Create an array of VARIABLE_MTRR_INFO structures for every
+MTRR range that you want to validate.  If any of the checked regions don't have the
+matching caching type the test will report an error for the failing range and return.

```

Comment:
I don't think the library interface should be tied to "Ready to Boot" as a whole.

Reply:
I recommend leaving this generic and let individual functions handle the boot point details.